src/platforms/ctxlink/platform.c

```diff
@@ -213,7 +330,17 @@ bool platform_nrst_get_val(void)
 
 const char *platform_target_voltage(void)
 {
-	return "Unknown";
+	static char target[64] = {0};
+	uint32_t val = platform_target_voltage_sense();
+	target[0] = '0' + val / 1000U;
+	target[1] = '.';
+	val = val % 1000U;
+	target[2] = '0' + val / 100U;
+	val = val % 100U;
+	target[3] = '0' + val / 10U;
+	target[4] = 'V';
+	strcat(target, platform_battery_voltage());
```

Comment:
Because of how the battery voltage part works, this may overflow the storage for this string. Please use `strncat()` for safety.

Reply:
Changed to use strncat.